include/gatekeeper_fib.h

```diff
@@ -165,6 +165,38 @@ struct gk_fib {
 	} u;
 };
 
+struct gk_fib_ether_entry {
+
+	bool     stale;
+
+	/* The IP address of the nexthop. */
+	char     nexthop_ip[INET6_ADDRSTRLEN];
+
+	uint16_t ether_type;
+
+	char     d_addr[ETHER_ADDR_FMT_SIZE];
+
+	char     s_addr[ETHER_ADDR_FMT_SIZE];
+
+	uint32_t ref_cnt;
+};
+
+struct gk_fib_dump_entry {
+
+	/* The prefix string. */
+	char     prefix[INET6_ADDRSTRLEN + 5];
```

Comment:
Add a comment here for the meaning of the magic number 5.

Reply:
Also, shouldn't it be 4? 1 for the "/" plus 3 for the prefix length which is at most three digits (in the case of IPv6).